### PR TITLE
Added CORE-V-MCU GCC/Newlib toolchain to IDE    

### PR DESCRIPTION
**Pull request by promodkumar-ashling**
Added CORE-V-MCU GCC/Newlib toolchain to IDE using Embedded CDT RISC-V Extension Point

### CODE REVIEW

File: releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target [hunk 1]
```diff
@@ -28,22 +27,37 @@
...
-			<unit id="ilg.gnumcueclipse.managedbuild.cross.arm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.managedbuild.cross.riscv.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.ad.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.cortexm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.freescale.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.sifive.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.stm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.packs.feature.feature.group" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/embed-cdt/releases/5.1.1/p2/"/>
+			<repository location="https://download.eclipse.org/embed-cdt/releases/5.1.3/p2/"/>
```
Comment (ruspl-afed) [inline releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target:30]: Let's do separate PR if we need to upgrade to this version

File: features/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Toolchain
+Bundle-SymbolicName: org.openhwgroup.corev.toolchain;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Automatic-Module-Name: org.openhwgroup.corev.toolchain
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: ilg.gnumcueclipse.managedbuild.cross.riscv;bundle-version="2.9.3"
```
Comment (ruspl-afed) [inline features/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:8]: Do we really need this version? Let's start from 0.0.0

File: releng/org.openhwgroup.corev.ide.aggregator/pom.xml [hunk 3]
```diff
@@ -34,6 +34,7 @@
 		<module>..\..\releng\org.openhwgroup.corev.ide.repository</module>
 
 		<module>..\..\products\org.openhwgroup.corev.ide.product</module>
+		<module>..\..\features\org.openhwgroup.corev.toolchain</module>
```
Comment (ruspl-afed) [inline releng/org.openhwgroup.corev.ide.aggregator/pom.xml:37]: Bundle should be placed to "bundles"

**Review by ruspl-afed — CHANGES_REQUESTED** [review]

File: releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target (hunk 1 above)
Comment (promodkumar-ashling) [reply releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target:30]: done

File: bundles/org.openhwgroup.corev.toolchain/build.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+source.. = src/
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/build.properties:1]: License header needed

File: bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Toolchain
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:3]: Name should be more descriptive

File: bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+Bundle-SymbolicName: org.openhwgroup.corev.toolchain;singleton:=true
+Bundle-Version: 0.0.0.qualifier
+Automatic-Module-Name: org.openhwgroup.corev.toolchain
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:6]: Bundle-Vendor should be specified

File: bundles/org.openhwgroup.corev.toolchain/plugin.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/plugin.xml:3]: License header should be added

File: releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target [hunk 8]
```diff
@@ -11,8 +11,7 @@
 
 	Contributors:
 		Alexander Fedorov (ArSysOp) - initial API and implementation
--->
-<target name="org.openhwgroup.corev.ide.target" sequenceNumber="1">
+--><target name="org.openhwgroup.corev.ide.target" sequenceNumber="1">
```
Comment (ruspl-afed) [inline releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target:14]: We already have .target updated and merged, this file should be removed from PR

File: bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Toolchain
+Bundle-SymbolicName: org.openhwgroup.corev.toolchain;singleton:=true
+Bundle-Version: 0.0.0.qualifier
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:5]: this should be 0.1.0.qualifier

File: bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
+Automatic-Module-Name: org.openhwgroup.corev.toolchain
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: ilg.gnumcueclipse.managedbuild.cross.riscv;bundle-version="2.9.3"
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:8]: And this _can_ be bundle-version="0.0.0" if we do not need some specific version

File: bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF [hunk 11]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Toolchain
+Bundle-SymbolicName: org.openhwgroup.corev.toolchain;singleton:=true
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.toolchain/META-INF/MANIFEST.MF:4]: I suppose bundle id could be more specific

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
